tools/workspace/abseil-cpp/repository.bzl

```diff
@@ -0,0 +1,10 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def abseil_cpp_repository(name = "abseil-cpp"):
+    version = "20240722.0"
```

Comment:
We really should have a mechanism to ensure that this is kept in sync with CMakeFiles...

Reply:
This is a good idea. Let me add that mechanism.